Makefile

```diff
@@ -4,6 +4,20 @@ image_tag := $(prog)
 image_tar := $(prog)-kaniko.tar
 image_eif := $(image_tar:%.tar=%.eif)
 
+.PHONY: build run start-enclave-service restart-enclave-service run-enclave start-gvproxy forward-ports create-instance setup-host list-enclaves
+
+
+# use kaniko to build the image
+# https://github.com/brave/star-randsrv/blob/main/Makefile
+build:
```

Comment:
I moved this to the top, so when people run make, it'll be the command that's ran